src/main/perf/facets/IndexFacets.java

```diff
@@ -124,4 +131,20 @@ public static void main(String args[]) throws IOException {
         AtomicInteger counter = new AtomicInteger(0);
         return item -> consumer.accept(counter.getAndIncrement(), item);
     }
+
+    private static void deleteDirectoryIfExists(Path path) throws IOException {
+        if (Files.notExists(path) == false) {
```

Comment:
Hmm maybe just `Files.exists` instead?

Reply:
Oops double negative. Changed in next revision.